platforms/extensibility/test-kit/src/main/java/org/gradle/testkit/runner/internal/ToolingApiGradleExecutor.java

```diff
@@ -260,9 +261,8 @@ public void statusChanged(ProgressEvent event) {
             }
         }
 
-        private boolean accept(TaskProgressEvent event) {
```

Comment:
small scope big name.

This is a perfect example from the clean code book, why not to do, the way it is right now...